scylla-gdb.py

```diff
@@ -4183,10 +4253,7 @@ def filename(sst):
         int_type = gdb.lookup_type('int')
         version_number = int(sst['_version'])
         version_name = list(version_to_format)[version_number]
-        try:
-            generation = sst['_generation']['_value']
-        except gdb.error:
-            generation = sst['_generation']
+        generation = sst['_generation']
```

Comment:
Will this work with old integer based generations? How about the middle ground where we had the generation struct, with a `_value` field. We try to keep `scylla-gdb.py` compatible with all currently supported versions. It is a loosing battle, but we keep fighting anyway.

Reply:
that’s the first case in the `to_string` implementation. Where the type is `int64_t`. I also explain this briefly in the commit message.